### PR TITLE
Networking Types & Characteristics Module closes #473

### LINKED ISSUE #473: Networking Types & Characteristics Module
possible tasks:
    +what are the key differences between LAN and WAN?
    +how can a network show each characteristic?
    +how can a network fail to implement each characteristic?

### PR DESCRIPTION
**Pull request by nickrstewarttds**

### CODE REVIEW

File: topics/networking/modules/types-&-characteristics/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+| **Low Latency** | **Error rate** | Lower; due to local connectivity | Higher; as a result of distribution |
+| **High Availability** | **Communication link** | 'In-house'; e.g. coaxial cables, fibre-optics, ethernet | External; e.g. phone lines, microwave links, satellite channels |
+| **Redundancy** | **Communication cost** | Lower; simpler operation allows for reliability | Higher; some aspects of operation may be leased or unreliable |
+| **Security** | **Ownership** | Single; usually a single organisation or individual | Multiple; essentially several interconnecting LANs |
+
+These differences help to explain the difference in scale between *LAN* and *WAN*:
+ - *LAN* is a small, interconnected collection of nodes, usually operated by a single entity, across a small area
+ - *WAN* connects several *LAN* together to allow for wider, though less reliable, communication across multiple entities
+
+## Tasks
```
Comment (tvaidotas) [inline topics/networking/modules/types-&-characteristics/README.md]: for the tasks could you provide expandable answers to the questions

File: topics/networking/modules/types-&-characteristics/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+Some networks might also require the usage of specific hardware or transmission standards.
+
+## Network Types
+
+The **type** of a network usually relies on how much physical space it covers.
+
+This makes it easy to explore the differences between the networks we use on a regular basis.
+
+The smallest networks area may only extend for a few nanometres, while the largest networks stretch over the entire planet.
```
Comment (JHarry444) [inline topics/networking/modules/types-&-characteristics/README.md]:
```suggestion
The smallest network's area may only extend for a few nanometres, while the largest networks stretch over the entire planet.
```

File: topics/networking/modules/types-&-characteristics/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+| **Security** | **Ownership** | Single; usually a single organisation or individual | Multiple; essentially several interconnecting LANs |
+
+These differences help to explain the difference in scale between *LAN* and *WAN*:
+
+- *LAN* is a small, interconnected collection of nodes, usually operated by a single entity, across a small area
+- *WAN* connects several *LAN* together to allow for wider, though less reliable, communication across multiple entities
+
+## Tutorial
+
+### How can a network exhibit good characteristics?
```
Comment (JHarry444) [inline topics/networking/modules/types-&-characteristics/README.md]: Move to exercise section

File: topics/networking/modules/types-&-characteristics/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+<b>High availability</b> - a larger network will need to keep a close eye on how reliable its infrastructure is, as wiring, routers, servers and so on will all need to be upgraded and tested to ensure their reliability.
+
+<b>Redundancy</b> - the larger a network gets, the larger the Operational Expenditure (OpEx) will become. However, a larger network may need to maintain multiple servers or nodes, so as to distribute the points of failure which may occur. A good network will need to keep a close eye on how much redundancy is too much.
+
+<b>Security</b> - all networks should keep a close eye on who owns data within it, who is authorised to access that data, and who is authenticated at any given time, as well as maintaining security standards in legal terms, such as [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation).
+</details>
+
+## Exercises
+
+### Think about LAN and WAN use cases
```
Comment (JHarry444) [inline topics/networking/modules/types-&-characteristics/README.md]: Provide solution.

**Pull request merged**